net/src/client/tcp/actors/jsonrpc.rs

```diff
@@ -14,73 +14,111 @@ use async_jsonrpc_client::{
 };
 use futures::StreamExt;
 use futures_util::compat::Compat01As03;
+use rand::seq::SliceRandom;
 use serde::Serialize;
-use serde_json::{value, Value};
+use serde_json::value;
+
+pub use serde_json::Value;
 
 use super::Error;
 
+const DEFAULT_BACKOFF_TIME_MILLIS: u64 = 250;
+
+struct Connection {
+    backoff: Duration,
```

Comment:
Where are you using the backoff?

Reply:
Here: https://github.com/witnet/witnet-rust/blob/47994bc46113633244e4222c5e09ea2af470a121/net/src/client/tcp/actors/jsonrpc.rs#L89-L98